book.json

```diff
@@ -1,3 +1,13 @@
 {
   "plugins": ["anchorjs", "richquotes", "mathjax"]
+  "pluginsConfig":
```

Comment:
You don't need these. `richquotes` is already enabled.

Reply:
Yes, I have withdrawn the changes